<!DOCTYPE HTML>
<html>

<head>
  <title>Kristoffer Sahlin's homepage</title>
  <meta name="description" content="website description" />
  <meta name="keywords" content="website keywords, website keywords" />
  <meta http-equiv="content-type" content="text/html; charset=windows-1252" />
  <link rel="stylesheet" type="text/css" href="style/style.css" />
  
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-56580014-1', 'auto');
  ga('send', 'pageview');

</script>
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of the text -->
          <h1><a href="index.html"><span class="logo_colour">Kristoffer Sahlin</span></a></h1>
          <h2>Bioinformatician interested in applications of statistical methods and algorithms to analyze data</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected page - to highlight which page you're on -->
          <li class="selected"><a href="index.html">Home</a></li>
          <li><a href="https://github.com/ksahlin/ksahlin.github.io/blob/master/files/ksahlin.pdf">CV</a></li>
          <li><a href="software.html">Software</a></li>
          <li><a href="teaching.html">Teaching</a></li>
          <!-- <li><a href="awards.html">Awards</a></li> -->
          <!-- <li><a href="blog.html">Blog</a></li> -->
          <li><a href="publications.html">Publications</a></li>
        </ul>
      </div>
    </div>
    <div id="content_header"></div>
    <div id="site_content">
      <div id="banner"></div>
	  <div id="sidebar_container">
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <!-- insert your sidebar items here -->
            <h3>Contact</h3>
            <form method="post" action="#" id="subscribe">
              <p style="padding: 0 0 9px 0;"><input class="search" type="text" name="email_address" value="ksahlin [at] kth [dot] se" onclick="javascript: document.forms['subscribe'].email_address.value=''" /></p>
            </form>
          </div>
          <div class="sidebar_base"></div>
        </div>
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <h3>Can be found on</h3>
            <ul>
              <li><h5> <a href="http://scholar.google.com/citations?user=IAGyYNEAAAAJ&hl=en">Google scholar</a></h5></li>
              <li><h5> <a href="https://github.com/ksahlin">GitHub</a></h5></li>  
              <li><h5> <a href="http://www.csc.kth.se/~ksahlin/">KTH</a></h5></li>          
            </ul>
          </div>
          <div class="sidebar_base"></div>
        </div>
      </div>
      <div id="content">
        <!-- insert the page content here -->
        <p>
          <h3>Bio</h3>
            I obtainedmy PhD in Computer science from <a href="http://www.kth.se/csc">School of Computer Science and Communication</a>, KTH Royal institute of Technology (Supervisor: <a href="http://www.nada.kth.se/~arve/">Lars Arvestad</a>). Before my PhD I earned a bachelor's degree in mathematics (<a href="http://www2.math.su.se/gemensamt/grund/exjobb/matte/2008/rep10/report.pdf">my Bachelor thesis</a> on theoretcal properties of splines) and a master's degree in mathematical statistics (<a href="http://www2.math.su.se/matstat/reports/master/2011/rep2/report.pdf">my Master thesis</a> on convergence and burn-in estimation of MCMC output) from Stockholm University, Sweden. I have had postdoctoral training at Penn State University with  <a href="http://medvedevgroup.com/">Paul Medvedev</a> as mentor (Sept 2015 - June 2019), and I am currently a postdoc at University of Helsinki in <a href="https://www.helsinki.fi/en/researchgroups/genome-scale-algorithmics/people/veli-makinen#section-54456">Veli MÃ¤kinen's</a> group. <br>

          <h3>Current research</h3>
            I use mathematics, modeling, algorithms and simulations to understand problems in biology. My main focus has been on developing algorithms to analyze DNA and RNA sequencing data for problems related to genome assembly, structural variation detection, and transcriptome analysis. Recent reasearch involves transcript reconstruction, isoform detection, and SNV calling in long-read sequencing data (<a href="https://www.nature.com/articles/s41467-018-06910-x">paperd</a>), and long-read clustering techniques (<a href="https://www.biorxiv.org/content/early/2018/11/06/463463">manuscript</a>). I have also worked a lot with genomic (short) read sequencing for genome assembly, e.g., for <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-281">scaffolding</a> and <a href="https://www.liebertpub.com/doi/pdf/10.1089/cmb.2016.0124">structural variation detection</a>. I'm looking into aproaches integrating both long and short as well as genomic and transcriptomic datasets for, e.g., intron and UTR variation detection, isoform specific expression analysis. <br>

            <!--    I work with method development for sequencing techniques but my focus shifted to transcriptomics and long-read sequencing data. I designed algorithms that combined graph theory with statistical methods to cluster and error-correct long reads from transcript sequencing \citep{Sahlin2018-oj,Sahlin2018-ol}.

            During my PhD I developed algorithms for genome assembly (scaffolding in particular) and Structural variation detection using Illumina paired end and mate-pair data. Recent advances in sequencing techniques <a href="http://biorxiv.org/content/early/2015/09/15/026468">listed in GIAB preprint</a>  provides data that, combined with post-processing algorithms, has the potential to create haplotype phased genome assemblies, detecting large and complex structural variations, and derive fully phased haplotype specific transcript isoforms. Work is still needed on the computational method development side to fully make use of the data that these techniques provide. My postdoctoral research focuses on this. Specifically I work with algorithm development within transcriptome analysis, structural variation and genome assembly where, in addition to Illumina paired-end/mate-pairs, I use sequencing data such as pacbio CCS reads, and synthetic long reads. --> 

          <h3>Motivation and interests</h3>
            I'm interested in applications of statistical and mathematical methods and algorithms to analyze data. I'm also strongly motivated by developing sustainable and easily accesibele open source software --- all my code can be found on GitHub.  

        </p>
      </div>
    </div>
    <div id="content_footer"></div>
    <div id="footer">
      <p><a href="index.html">Home</a> | <a href="software.html">Software</a> | <a href="teaching.html">Teaching</a> | <a href="publications.html">Publications</a></p>
      <p>Copyright &copy; simplestyle_banner | <a href="http://validator.w3.org/check?uri=referer">HTML5</a> | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a> | <a href="http://www.html5webtemplates.co.uk">design from HTML5webtemplates.co.uk</a></p>
    </div>
  </div>
</body>
</html>
